### PR TITLE
Resolve found module but no type hints or library stubs

### PR DESCRIPTION
**Pull request by yiboliu**
### Reasons for making this change

This commit resolves all issues annotated as 'found module but no type hints or library stubs' by adding `# type: ignore`. Ref: https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
### Related issues
https://github.com/codalab/codalab-worksheets/issues/2080

### Checklist

* [ ] I've added a screenshot of the changes, if this is a frontend change
* [ ] I've added and/or updated tests, if this is a backend change
* [ ] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [ ] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

**Review by epicfaace — COMMENTED** [review]
Could we instead use the approach in which we specify ignored libraries in a mypy config file and use the `ignore_missing_imports` option (see https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-type-hints-for-third-party-library)?

I think that will be easier for us to keep track of to keep all these rules in a single file (such as eslint ignored rules) rather than having them set in comments whenever the libraries are imported.

**Comment by yiboliu** [comment]
> Could we instead use the approach in which we specify ignored libraries in a mypy config file and use the `ignore_missing_imports` option (see https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-type-hints-for-third-party-library)?
> 
> I think that will be easier for us to keep track of to keep all these rules in a single file (such as eslint ignored rules) rather than having them set in comments whenever the libraries are imported.

To add to a config file, do we have a default one? Or I can create one myself? https://mypy.readthedocs.io/en/latest/config_file.html#config-file

**Comment by percyliang** [comment]
Could you add a call in the `pre-commit.sh` script?  Are there no warnings now?  If so, we should also add it to CI.

**Comment by yiboliu** [comment]
> Could you add a call in the `pre-commit.sh` script? Are there no warnings now? If so, we should also add it to CI.

No warnings about `found module but no type hints or library stubs`. I'll fix other issues in following commits.

File: pre-commit.sh [hunk 1]
```diff
@@ -19,3 +19,6 @@ venv/bin/mkdocs build || { rm -rf venv; exit 1; }  # Outputs to `site`
 # Fix Python and JavaScript style (mutates code!)
 venv/bin/black codalab scripts *.py || exit
 npm run --prefix frontend format
+
+#Check about mypy warnings
```
Comment (percyliang) [inline pre-commit.sh]: nit: space between '#' and 'Check'

File: pre-commit.sh [hunk 2]
```diff
@@ -19,3 +19,6 @@ venv/bin/mkdocs build || { rm -rf venv; exit 1; }  # Outputs to `site`
 # Fix Python and JavaScript style (mutates code!)
 venv/bin/black codalab scripts *.py || exit
 npm run --prefix frontend format
+
+# Check about mypy warnings
+mypy codalab/model/bundle_model.py
```
Comment (percyliang) [inline pre-commit.sh]: Make sure lines end with newline

**Comment by teetone** [comment]
Will the other classes mentioned in the issue https://github.com/codalab/codalab-worksheets/issues/2080 be handled in another pull request?

**Review by teetone — APPROVED** [review]

**Comment by yiboliu** [comment]
> Will the other classes mentioned in the issue #2080 be handled in another pull request?

Actually I used `mypy .` to identify problems, so all classes are resolved.

**Pull request merged**